Onboarding/Connector/Scripts/Documentation/BashScriptTsgAndFaq.md

```diff
@@ -0,0 +1,90 @@
+# For the bash script
+
+## FAQ
+
+### How do I run the script?
+
+Use the below commands
+
+For single subscription
+
+```bash
+chmod +x create-impact-reporting-connector.sh
+./create-impact-reporting-connector.sh --subscription-id <subscription_id>
+```
+
+For new line separated subscription list in a file
+
+```bash
+chmod +x create-impact-reporting-connector.sh
+./create-impact-reporting-connector.sh --file-path <file_path>
+```
+
+### Can I run this script for multiple subscriptions at once?
+
+Yes, use the `--file-path` option with a file containing newline-separated subscription IDs.
+
+### What permissions are needed to run this script?
+
+You need to have **Contributor** permission to log in to Azure, register resource providers, and create connectors in the Azure subscriptions. You also need to have either of **'Owner'**, **'User Access Administrator'** and **'Privileged Role Administrator'** permissions to create and assign custom roles.
```

Comment:
For registering the RPs and creating connectors, Owner should also work. So we should mention Contributor or Owner permission so that they'll know that Owner will suffice for both steps.